pkgs/by-name/eb/eb-garamond/package.nix

```diff
@@ -1,23 +1,40 @@
 {
   lib,
   stdenvNoCC,
-  fetchzip,
+  fetchFromGitHub,
+  python3,
+  ttfautohint,
 }:
-
 stdenvNoCC.mkDerivation rec {
   pname = "eb-garamond";
   version = "0.016";
 
-  src = fetchzip {
-    url = "https://bitbucket.org/georgd/eb-garamond/downloads/EBGaramond-${version}.zip";
-    hash = "sha256-P2VCLcqcMBBoTDJyRLP9vlHI+jE0EqPjPziN2MJbgEg=";
+  src = fetchFromGitHub {
+    owner = "georgd";
+    repo = "EB-Garamond";
+    tag = "v${version}";
+    hash = "sha256-3eByiJ0gVpKXtKP8AH0P9G5vTbdvkmnfD4Dn2JQ7K54=";
   };
 
+  nativeBuildInputs = [
+    (python3.withPackages (p: [ p.fontforge ]))
+    ttfautohint
+  ];
+
+  buildPhase = ''
+    runHook preBuild
+    make TAG_COMMIT=${src.rev} TAG=nightly COMMIT=${src.rev} DATE=19700101
+    runHook postBuild
+  '';
+
   installPhase = ''
     runHook preInstall
 
-    install -Dm644 otf/*.otf                                 -t $out/share/fonts/opentype
-    install -Dm644 Changes README.markdown README.xelualatex -t $out/share/doc/${pname}-${version}
+    install -Dm644 build/*.ttf  -t $out/share/fonts/truetype/EBGaramond
+    install -Dm644 build/*.otf  -t $out/share/fonts/opentype/EBGaramond
+    install -Dm644 web/*.woff   -t $out/share/fonts/woff/EBGaramond
```

Comment:
I think these should be without the additional `/EBGaramond` directory (cf. other font packages such as [dejavu](https://github.com/NixOS/nixpkgs/blob/107d5ef05c0b1119749e381451389eded30fb0d5/pkgs/data/fonts/iosevka/default.nix#L126-L128), [iosevka](https://github.com/NixOS/nixpkgs/blob/5df43628fdf08d642be8ba5b3625a6c70731c19c/pkgs/by-name/io/iosevka/package.nix#L129-L131), [garamond-libre](https://github.com/NixOS/nixpkgs/blob/5df43628fdf08d642be8ba5b3625a6c70731c19c/pkgs/by-name/ga/garamond-libre/package.nix#L20))